tests/unit/parallel_test.py

```diff
@@ -148,6 +148,7 @@ def test_parallel_execute_alignment(capsys):
         func=lambda x: x,
         get_name=six.text_type,
         msg="Aligning",
+        use_existing_writer=False
```

Comment:
Is this parameter only used for tests? If so, wouldn't it make more sense to get rid of it and reset `ParallelStreamWriter.instance` to `None` in the `setUp` method?